tests/manual-test-cases/Group5-Functional-Tests/5-4-High-Availability.robot

```diff
@@ -232,15 +232,19 @@ Test
     \     Should Be Equal As Integers  ${rc}  0
     \     Should Be Equal  ${output}  running
     \     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} rm -f ${c}
-    \     Should Be Equal As Integers  ${rc}  0
+    \     Run Keyword If  ${rc} != 0  Should Not Contain  ${output}  ESX host is temporarily disconnected.
+    \     Should Be Equal As Integer  ${rc}  0
+    \     Log To Console  ${output}
```

Comment:
We should place the `Log To Console` before the return code check so we can see the message in case the exit code is non-zero.